modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/PersistentPageMemoryMvPartitionStorage.java

```diff
@@ -215,7 +242,25 @@ public RaftGroupConfiguration committedGroupConfiguration() {
         }
 
         try {
-            return groupConfigFromBytes(meta.lastGroupConfig());
+            groupConfigReadWriteLock.readLock().lock();
```

Comment:
Forgot
```
if (!closeBusyLock.enterBusy()) {
            throwStorageClosedException();
        }
```

Reply:
We already have busy lock taken, in line 240. Isn't this enough?

Reply:
my fault